smartype-generator/src/main/kotlin/com/mparticle/smartype/generator/Generator.kt

```diff
@@ -187,33 +187,38 @@ class Generate : CliktCommand(name="generate", help = "Generate Smartype Client
         }
 
         if (options.webOptions.enabled) {
-            val webBuildDirectory = File(projectDirectory).resolve("smartype/build/distributions")
-            val smartypeBuildDir = File(projectDirectory).resolve("build")
-            if (webBuildDirectory.exists()) {
-                val mvWeb =
-                    listOf(
-                        "mv",
-                        webBuildDirectory.absolutePath,
-                        File(binOutputDirectory).resolve("web").absolutePath
-                    )
-                val pb5 = ProcessBuilder(mvWeb)
-                pb5.redirectOutput(ProcessBuilder.Redirect.INHERIT)
-                pb5.redirectError(ProcessBuilder.Redirect.INHERIT)
-                val p5 = pb5.start()
-                p5.waitFor();
+            val typescriptDefBuildDir = File(projectDirectory).resolve("build")
```

Comment:
they decided to change where JS outputs distribution files to in Kotlin 1.9, so we need to copy them from a different place to the final output